runtime/shiden/src/lib.rs

```diff
@@ -1259,16 +1264,53 @@ pub type Executive = frame_executive::Executive<
     Migrations,
 >;
 
+pub struct StaticTierParamsMigration;
+impl OnRuntimeUpgrade for StaticTierParamsMigration {
+    fn on_runtime_upgrade() -> Weight {
+        let onchain_storage_version =
+            pallet_dapp_staking_v3::Pallet::<Runtime>::on_chain_storage_version();
+
+        if onchain_storage_version == 8 {
+            let tier_thresholds = BoundedVec::try_from(vec![
+                TierThreshold::DynamicPercentage {
+                    percentage: Perbill::from_parts(35_700_000), // 3.57%
+                    minimum_required_percentage: Perbill::from_parts(23_800_000), // 2.38%
+                },
+                TierThreshold::DynamicPercentage {
+                    percentage: Perbill::from_parts(8_900_000), // 0.89%
+                    minimum_required_percentage: Perbill::from_parts(6_000_000), // 0.6%
+                },
+                TierThreshold::DynamicPercentage {
+                    percentage: Perbill::from_parts(23_800_000), // 2.38%
+                    minimum_required_percentage: Perbill::from_parts(17_900_000), // 1.79%
+                },
+                TierThreshold::FixedPercentage {
+                    required_percentage: Perbill::from_parts(6_000_000), // 0.6%
+                },
+            ])
+            .unwrap();
+
+            pallet_dapp_staking_v3::StaticTierParams::<Runtime>::mutate(|config| {
+                config.tier_thresholds = tier_thresholds;
+            });
+
+            <Runtime as frame_system::Config>::DbWeight::get().reads_writes(1, 1)
+        } else {
+            Weight::zero()
+        }
+    }
+}
+
 /// All migrations that will run on the next runtime upgrade.
 ///
 /// Once done, migrations should be removed from the tuple.
 pub type Migrations = (
     cumulus_pallet_xcmp_queue::migration::v4::MigrationToV4<Runtime>,
     // permanent migration, do not remove
     pallet_xcm::migration::MigrateToLatestXcmVersion<Runtime>,
-    // XCM V3 -> V4
-    pallet_xc_asset_config::migrations::versioned::V2ToV3<Runtime>,
-    pallet_identity::migration::versioned::V0ToV1<Runtime, 250>,
```

Comment:
I'd suggest to keep this in Shiden/Astar until we actually do the upgrade with this code.